### DISCUSSION TITLE
What to do if netlify isn't recognizing your routes

### DISCUSSION THREAD

**Discussion by katherinevfry** [nss-evening-cohort-14/evening-client-side#248] [Look at what I learned!]
After deploying my bio site to Netlify, I was getting a 404 error when I tried to access my authed route.

After lots of googling and reading through the netlify docs, I came across this solution:

add a file called `netlify.toml` to your root directory. (_toml_ is short for _Tom's Obvious, Minimal Language_. the more you know.)

add the following lines of code:
```
[[redirects]]
  from = "/*"
  to = "/index.html"
  status = 200
```
push up and redeploy! There might be better ways to do this, but this is what worked for me.

**Comment by drteresavasquez** [comment]
Excellent find! And way to step out and debug your issue!

**Comment by hmparsons-sde** [comment]
Omg Katy! This helped me so much!

**Comment by cmeffley** [comment]
I had the same issue, THANK YOU!

**Comment by Gabrielle-Tobermann** [comment]
THANK YOU KATY

**Comment by caseywalker** [comment]
Iron Katy!

**Reply by katherinevfry** [reply]
🦸‍♀️

**Comment by drteresavasquez** [comment]
Also, if the toml file does not work, remove it and in your public directory, create a file called `_redirects` and add this line:
`/*    /index.html   200`
